From 31401d2b348daa104d7105e5311d30503f8c4b44 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 28 Apr 2021 05:28:35 -0700
Subject: [PATCH 1504/1921] Revert "net: mvpp2: Add NET_DSA select to MVPP2
 config"

This reverts commit 7a160820cbb5761d2288e394337bf19d411ad2d9.

Reason for revert: patch create unmet direct dependencies detected for NET_DSA

Change-Id: Ica3da8222e5f69c7c0c8ed3fb150b057fce5d8f9
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50785
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Tested-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/Kconfig b/drivers/net/ethernet/marvell/Kconfig
index 62b133fa3856..fb942167ee54 100644
--- a/drivers/net/ethernet/marvell/Kconfig
+++ b/drivers/net/ethernet/marvell/Kconfig
@@ -86,7 +86,6 @@ config MVPP2
 	depends on ARCH_MVEBU || COMPILE_TEST
 	select MVMDIO
 	select PHYLINK
-	select NET_DSA
 	---help---
 	  This driver supports the network interface units in the
 	  Marvell ARMADA 375, 7K and 8K SoCs.
-- 
2.31.1

